### ISSUE TITLE
Project Section in README - should be divide to subject

### ISSUE THREAD

**Issue by JonathanDn** [lirantal/awesome-opensource-israel#42]
Would be easier to navigate the list now that Projects is quite a long list if it would be divided to subjects and fields of interest.

**Comment by lirantal** [comment]
Agree, though let's allow the project list to grow a bit and learn what categories we can build before we prematurely optimize.

**Comment by shlomif** [comment]
Hi,

On Thu, 27 Dec 2018 07:23:09 -0800
Jonathan Doron <notifications@github.com> wrote:

> Maybe its good to divide to more generic "macro" subject for now?
> Frontend, Backend, Util
> 

Not all the projects on the list are web-based or web-centric. Also see
https://www.shlomifish.org/open-source/resources/israel/list-of-projects/ .


-- 
-----------------------------------------------------------------
Shlomi Fish       http://www.shlomifish.org/
The Human Hacking Field Guide - http://shlom.in/hhfg

I hope that if it had not been clear before, it isn’t less clear now.
    — One of Shlomi Fish’s Technion Lecturers

Please reply to list if it's a mailing list post - http://shlom.in/reply .

**Comment by JonathanDn** [comment]
Than we can also have a Misc area
